File: AdminPanal/AdminPanal.Web/Modules/AylaStoreModule/ContactUs/ContactUsForm.cs
using Serenity.ComponentModel;

namespace AdminPanal.AylaStoreModule.Forms;

[FormScript("AylaStoreModule.ContactUs")]
[BasedOnRow(typeof(ContactUsRow), CheckNames = true)]
public class ContactUsForm
{
    public string Name { get; set; }
    public string Email { get; set; }
    public string Subject { get; set; }
    public string Description { get; set; }
}

File: AdminPanal/AdminPanal.Web/Modules/ServerTypes/AylaStoreModule/ContactUsForm.ts
import { StringEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface ContactUsForm {
    Name: StringEditor;
    Email: StringEditor;
    Subject: StringEditor;
    Description: StringEditor;
}

export class ContactUsForm extends PrefixedContext {
    static formKey = 'AylaStoreModule.ContactUs';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!ContactUsForm.init)  {
            ContactUsForm.init = true;

            var w0 = StringEditor;

            initFormType(ContactUsForm, [
                'Name', w0,
                'Email', w0,
                'Subject', w0,
                'Description', w0
            ]);
        }
    }
}